### PR TITLE
Fix typos in programming guide and mobile gaming example

### PR DESCRIPTION
**Pull request by robinyqiu**
http://apache-beam-website-pull-requests.storage.googleapis.com/491/index.html

### CODE REVIEW

File: src/documentation/programming-guide.md [hunk 1]
```diff
@@ -1072,7 +1072,7 @@ If your input `PCollection` uses the default global windowing, the default
 behavior is to return a `PCollection` containing one item. That item's value
 comes from the accumulator in the combine function that you specified when
 applying `Combine`. For example, the Beam provided sum combine function returns
-a zero value (the sum of an empty input), while the max combine function returns
+a zero value (the sum of an empty input), while the min combine function returns
```
Comment (lukecwik) [inline src/documentation/programming-guide.md]: `min` returns a maximal `value`?

Don't you want to leave this at `max`?

Comment (robinyqiu) [reply src/documentation/programming-guide.md]: It should be `min`, because the identity of `min` is `maximal value`.

**Comment by asfgit** [comment]
Error: PR failed in verification; check the Jenkins job for more information.

**Comment by lukecwik** [comment]
This was merged.

**Comment by robinyqiu** [comment]
The changes don't seem to be there on the website now. Did it get merged?

**Comment by lukecwik** [comment]
Next merge that rebuilt the website got this update.

**Pull request closed without merging**